src/rust-crypto/CrossSigningIdentity.ts

```diff
@@ -87,12 +87,18 @@ export class CrossSigningIdentity {
                     "bootstrapCrossSigning: Cross-signing private keys not found locally, but they are available " +
                         "in secret storage, reading storage and caching locally",
                 );
-                await this.olmMachine.importCrossSigningKeys(
+                const status: CrossSigningStatus = await this.olmMachine.importCrossSigningKeys(
```

Comment:
Can we type this in the .d.ts file instead of needing it on every call? a la https://github.com/matrix-org/matrix-js-sdk/blob/develop/src/%40types/matrix-sdk-crypto-wasm.d.ts#L22

Reply:
I'm asking for the local d.ts file I linked to be updated @richvdh - not a file in another project.